docs_user/modules/proc_deploying-the-bare-metal-provisioning-service.adoc

```diff
@@ -1,45 +1,42 @@
-must already be imported into the control plane MariaDB, {identity_service_first_ref}, {networking_first_ref}, {image_service_first_ref}, and {block_storage_first_ref} should be in an operational state. Ideally, {compute_service_first_ref} has not been adopted yet if {bare_metal} is leveraged in a Bare Metal as a Service configuration.
-
-* Before deploying {rhos_prev_long} in {rhos_long}, you must ensure that the networks are ready, that you have decided the node selection, and also make sure any necessary changes to the {rhos_acro} nodes have been made. For {bare_metal} conductor services, it is necessary that the services be able to reach Baseboard Management Controllers of hardware which is configured to be managed by {bare_metal}. If this hardware is unreachable, the nodes may enter "maintenance" state and be unable to be acted upon until connectivity is restored at a later point in time.
-
-* You need the contents of `ironic.conf` file. Download the file so that you can access it locally:
+* You have imported the service databases into the control plane MariaDB. 
+* The {identity_service_first_ref}, {networking_first_ref}, {image_service_first_ref}, and {block_storage_first_ref} are operational.
++
+[NOTE]
+If you use the {bare_metal} in a Bare Metal as a Service configuration, you have not yet adopted the {compute_service_first_ref}.
+* For the {bare_metal} conductor services, the services must be able to reach Baseboard Management Controllers of hardware that is configured to be managed by the {bare_metal}. If this hardware is unreachable, the nodes might enter "maintenance" state and be unavailable until connectivity is restored later.
+* You have downloaded the `ironic.conf` file locally:
 +
 ----
 $CONTROLLER1_SSH cat /var/lib/config-data/puppet-generated/ironic/etc/ironic/ironic.conf > ironic.conf
 ----
 +
 [NOTE]
-It is critical that this configuration file comes from one of the controllers and not a {OpenStackPreviousInstaller} undercloud node. The {OpenStackPreviousInstaller} undercloud node specifically operated with different configuration which would not be appropriate or applicable to apply when adopting the Overcloud Ironic deployment.
-//kgilliga: What is meant by "overcloud Ironic deployment? Can this be changed to the "RHOSP Bare Metal Provisioning service deployment"?
-
-* If adopting the Ironic Inspector service you need the value of the `IronicInspectorSubnets` {OpenStackPreviousInstaller} parameter. Use the same values to populate the `dhcpRanges` parameter in the target environment.
-* Define the following shell variables. The values that are used are examples. Replace these example values with values that are correct for your environment:
+This configuration file must come from one of the Controller nodes and not a {OpenStackPreviousInstaller} undercloud node. The {OpenStackPreviousInstaller} undercloud node operates with different configuration that does not apply when you adopt the Overcloud Ironic deployment.
```

Comment:
Can "Overcloud Ironic" deployment be changed to the "RHOSP Bare Metal Provisioning service deployment"?